lib/Auth/Process/privacyidea.php

```diff
@@ -83,6 +84,43 @@ public function process(&$state)
 		}
 
 		if($piEnabled) {
+			if ($this->serverconfig['doTriggerChallenge']) {
+				$authToken = sspmod_privacyidea_Auth_utils::fetchAuthToken($this->serverconfig);
```

Comment:
I am not sure if it is necessary to split the service accounts of enrollment and triggerchallenge.
However, in both cases it is a pi-serviceaccount for simplesaml and it has the necessary rights assigned.

Reply:
It could be, that one has the right for trigger challenge and not for token init, couldn't it?
But apart from this it is necessary in that way, because it can be configured without token enrollment so we will do this only in this file.

Reply:
Yes it could. But again, the question is, if it would make sense to have two service accounts anyways. So this implementation is totally fine!